projects/opendr_ws/src/perception/scripts/point_cloud_dataset.py

```diff
@@ -13,6 +13,7 @@
 # See the License for the specific language governing permissions and
```

Comment:
It's probaby a cuda-side error. It uses preprocessing from Voxel-based 3D Det when preparing the dataset.

Reply:
I tested it with conda, python==3.8, 
numba                    0.53.0
numpy                    1.21.2
torch                    1.7.1
Cuda compilation tools, release 11.6, V11.6.55
Build cuda_11.6.r11.6/compiler.30794723_0

If it fails, 3D detection tests should also fail on your PC.

Reply:
Alright thanks, i'll look into testing the node locally. 

In the [dependencies.ini torch is set as version 1.9.0](https://github.com/opendr-eu/opendr/blob/0af383d1b8ae07cf0e48a26c370465e1ef3ccc60/src/opendr/perception/object_detection_3d/voxel_object_detection_3d/dependencies.ini#L5) and not 1.7.1, do you think that this might cause any issues?

Reply:
I think that our CI tests use torch==1.9, so it should be ok. I have OpenDR on different devices, so they may have different torch versions installed